pages/linux/flameshot.md

```diff
@@ -0,0 +1,25 @@
+# flameshot 
+
+> Screenshot utility with a gui interface.
+> Supports basic image editing, such as text, shapes, colors, and imgur.
+> More information: <hhttps://flameshot.js.org/>.
```

Comment:
```suggestion
> More information: <https://flameshot.js.org>.
```

Reply:
@navarroaxel  https://github.com/tldr-pages/tldr-bot/issues/10

Reply:
We have a plan for a new linter, the current one is hard to maintain.